manim/mobject/coordinate_systems.py

```diff
@@ -225,6 +225,42 @@ def get_axis_labels(
         )
         return self.axis_labels
 
+    def add_coordinates(
+        self, *axes_numbers: Optional[Iterable[float]], **kwargs
+    ) -> VGroup:
+        """Adds labels to the axes.
+
+        axes_numbers
+            The the numbers to be added to the axes. Use ``None`` to represent an axis without labels.
+
+        Examples
+        --------
+
+        .. code-block:: python
+
+            ax = ThreeDAxes()
+            x_labels = range(1, 11)
+            z_labels = range(2, 12, 2)
+            ax.add_coordinates(x_labels, None, z_labels)  # x & z labels
```

Comment:
This also adds the default y-labels
```suggestion
            ax.add_coordinates(x_labels, None, z_labels)  # default y labels, custom x & z labels
```